challenges/objects-arrays.js

```diff
@@ -89,6 +137,14 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+// , "scientific_name: element.scientific_name"
```

Comment:
you could have used an interpolated string to return all needed info as a single object